Tests/UITests/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="NUnit" version="2.6.4" targetFramework="net45" />
-  <package id="Xamarin.UITest" version="2.0.3" targetFramework="net45" />
+  <package id="Xamarin.UITest" version="2.2.1" targetFramework="net45" />
```

Comment:
isn't that the package that we had to pin version?

Reply:
I don't remember that conversation. But now if we use 2.0.3 it fails and says to use the newer version.